frontend/src/Commons/Forms/FormHeader/FormHeader.style.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @desc Styling with BC Design Tokens and Emotion for Farm Information Input Module
+ * @author @GDamaso
+ */
+import styled from '@emotion/styled';
+import * as tokens from '@bcgov/design-tokens/js';
+
+const StyledFormHeader = styled.button<{ active: boolean }>`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-height: 51px;
+  max-width: 1311px;
+  height: 100%;
+  width: 100%;
+  color: ${tokens.typographyColorPrimary};
+
+  border-radius: 2px;
+  border-bottom-left-radius: ${({ active }) => (active ? '0' : '2px')};
+  border-bottom-right-radius: ${({ active }) => (active ? '0' : '2px')};
+  border-bottom: ${({ active }) => (active ? '0px' : '0px solid black')};
```

Comment:
Both sides of the ternary specify 0px? 
Side note: If using 0 as a value you don't need to specify units, since it is universal